src/testUtils.js

```diff
@@ -33,19 +34,42 @@ function applyTransform(module, options, input, testOptions = {}) {
     options || {}
   );
 
+  if (isPromise(output)) {
+    return output.then(result => {
+      return (result || '').trim();
+    });
+  }
+
   return (output || '').trim();
 }
 exports.applyTransform = applyTransform;
 
 function runSnapshotTest(module, options, input) {
   const output = applyTransform(module, options, input);
+
+  if (isPromise(output)) {
```

Comment:
Too bad we can't use async here, because then this could be a bit cleaner:

```
let output = applyTransform(.......);
if (isPromise(output)) {
  output = await output;
}
expect(output).toMatchSnapshot();
```

Reply:
Yes, I think so. On the other hand, using async/await will cause the return value to become a promise, I'm not sure if this will affect the existing logic...
I think these modifications are ugly, i'll close the PR.